lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -84,7 +84,8 @@ function dosomething_northstar_register_user($form_state) {
 function _dosomething_northstar_build_http_client() {
   $base_url = NORTHSTAR_URL;
   if (getenv('DS_ENVIRONMENT') === 'local') {
-      $base_url .=  ":" . NORTHSTAR_PORT;
+      // $base_url .=  ":" . NORTHSTAR_PORT;
+    $base_url = 'http://northstar.dev:' . NORTHSTAR_PORT;
```

Comment:
I actually changed this back - right now, the $base_url prints `http://northstar.dev:8000` where as before, it printed `http://northstar-qa.dosomething.org:8000`. Is this ok?